File: pytyon_study/Day2_3.py
age = int(input("나이를 입력하세요: "))

if age >= 20:
    print("성인입니다.")
elif age >= 17:
    print("고등학생입니다.")
elif age >= 14:
    print("중학생입니다.")
elif age >= 8:
    print("초등학생입니다.")
else:
    print("어린이입니다.")

File: java_study/src/Day2_3.java
import java.util.Scanner;

public class Day2_3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("나이를 입력하세요: ");
        int age = sc.nextInt();

        if (age >= 20) {
            System.out.println("성인입니다.");
        }
        else if (age >= 17) {
            System.out.println("고등학생입니다.");
        }
        else if (age >= 14) {
            System.out.println("중학생입니다.");
        }
        else if (age >= 8) {
            System.out.println("초등학생입니다.");
        }
        else {
            System.out.println("어린이입니다.");
        }
    }
}
